apps/paf/behaviours/person-number.js

```diff
@@ -0,0 +1,41 @@
+const _ = require("lodash");
+
```

Comment:
is it worth explaining why you need this behaviour? i.e. if the options are optional we need to display the person number for each person

Reply:
comment added explaining behaviour